helper/schema/resource.go

```diff
@@ -184,6 +184,11 @@ type Resource struct {
 	// actions (Create, Read, Update, Delete, Default) to the Resource struct, and
 	// accessing them in the matching methods.
 	Timeouts *ResourceTimeout
+
+	// Description is used as the description for docs, the language server and
+	// other user facing usage. It can be plain-text or markdown depending on the
+	// global DescriptionKind setting.
```

Comment:
does the DescriptionKind setting need to be global? I imagine someone may want to set it on a resource by resource basis.

Reply:
I thought about this for a bit, I think considering that plaintext is valid markdown, its pretty safe to make it a single setting right now.